### DISCUSSION TITLE
OPENGWAS_JWT=<token> needs to be set in your .Renviron file. You can obtain a token from https://api.opengwas.io

### DISCUSSION THREAD

**Discussion by yanhuayizha** [MRCIEU/TwoSampleMR#505] [General]
OPENGWAS_JWT=<token> needs to be set in your .Renviron file. You can obtain a token from https://api.opengwas.io
Can someone good soul show me how to set the token?

**Comment by explodecomputer** [comment]
Please update to the latest version of ieugwasr using `remotes::install_github("mrcieu/ieugwasr")` thanks

**Reply by yanhuayizha** [reply]
thank you very much
